shared/actions/fs/index.js

```diff
@@ -354,7 +355,8 @@ const getWaitDuration = (endEstimate: ?number, lower: number, upper: number): nu
 }
 
 let polling = false
-function* pollSyncStatusUntilDone(): Saga.SagaGenerator<any, any> {
+function* pollSyncStatusUntilDone(action: FsGen.FsActivityPayload): Saga.SagaGenerator<any, any> {
+  console.log('pollllllll')
```

Comment:
:fire:

Reply:
Ah; I only realized in my follow-up PR, so removed it there: https://github.com/keybase/client/pull/13391

Reply:
Actually I'll remove it here so it's never in master. Thanks